stake-pool/cli/src/output.rs

```diff
@@ -0,0 +1,179 @@
+use std::fmt::{Display, Formatter};
+use solana_cli_output::{QuietDisplay, VerboseDisplay};
+use solana_sdk::pubkey::Pubkey;
+use solana_sdk::stake::state::Lockup;
+use serde::{Serialize, Deserialize};
+use spl_stake_pool::state::{Fee, StakePool, StakeStatus, ValidatorList, ValidatorStakeInfo};
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CliStakePools {
+    pub pools: Vec<CliStakePool>
+}
+
+impl Display for CliStakePools {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        for pool in &self.pools {
+            writeln!(f,
+                     "Address: {}\tManager: {}\tLamports: {}\tPool tokens: {}\tValidators: {}",
+                     pool.address.to_string(),
+                     pool.manager.to_string(),
+                     pool.total_lamports.to_string(),
+                     pool.pool_token_supply.to_string(),
+                     pool.validator_list.len().to_string()
```

Comment:
I don't think you need any of these `to_string()` calls, as all these types implement Display

Reply:
Yes Thx I remove it